tests/cmds/test_profile.py

```diff
@@ -254,6 +254,34 @@ def test_update_profile_updates_existing_profile(
     )
 
 
+def test_update_profile_updates_default_profile(
+    runner, mock_cliprofile_namespace, user_agreement, valid_connection, profile
+):
+    name = "foo"
+    profile.name = name
+    mock_cliprofile_namespace.get_profile.return_value = profile
+    runner.invoke(
+        cli, ["profile", "update", "-s", "bar", "-u", "baz", "--disable-ssl-errors",],
```

Comment:
here and only line 278 removing the last `,` before the closing `]` should resolve the style errors.

Reply:
Thank you for pointing that out, I didn't realize the style corrections weren't fully automated.

Reply:
Thanks Alan